README.md

```diff
@@ -19,7 +28,22 @@ Usage
 To enable the `vimeo` xblock in your course:
 
 1. In Studio, go to `Settings > Advanced Settings`.
-1. Locate the `Advanced Module List` field, and add `"vimeo"` to that list.
-1. Add a Unit to your course, and click on the `Advanced` button to add a `Vimeo` component.
-1. Click `Edit` to add your Video URL, located on [vimeo.com](https://vimeo.com).
-1. Click Save to add the video to your course.
+2. Locate the `Advanced Module List` field, and add `"vimeo"` to that list.
+3. Add a Unit to your course, and click on the `Advanced` button to add a `Vimeo` component.
+4. Click `Edit` to add your Video URL, located on [vimeo.com](https://vimeo.com).
+5. Click Save to add the video to your course.
+
+Testing
+-----
+
+1. Make sure you have `tox` installed.
+2. Run the tox tests using the `tox` command.
+
+Installing on DevStack
+-----
+
+1. Start your *edX devstack*
+2. Once the *edX devstack* is up, run `make studio-shell`
+3. `source ../venvs/edxapp/bin/activate`
+4. `sudo -u edxapp /edx/bin/pip.edxapp install "git+https://github.com/open-craft/xblock-vimeo@nizar/juniper_upgrade#egg=xblock-vimeo"`
+5. `docker-compose restart studio`
```

Comment:
Nit: The xblock will need to be installed in lms as well otherwise the unit will fail to load in lms.

Reply:
Oh, fair enough, I completely missed that. I tested the xblock on the Studio mainly without trying it on the LMS. I'll add a line for that.